lag/lag.yaml

```diff
@@ -29,20 +43,26 @@ components:
         ethernet:
           $ref: '../device/ethernet.yaml#/components/schemas/Device.EthernetBase'
 
-    Lag.Protocol:
+    Lag.Ports.Protocol:
       type: object
       properties:
         choice:
           description: >-
-            The type of LAG protocol.
+            The type of controlling protocol for the LAG (ports group).
           type: string
           enum: [lacp, static]
           default: lacp
         lacp:
-          $ref: '#/components/schemas/Lag.Lacp'
+          $ref: '#/components/schemas/Lag.Ports.Lacp'
         static:
           $ref: '#/components/schemas/Lag.Static'
 
+    Lag.Protocol:
```

Comment:
Since the lag.port.protocol is now only a container for the lacp container should this be removed and the lacp container moved up to lag.port.lacp with the lacp container being optional?

Reply:
Done. The only concern, if we need to add similar protocols in the future (say MACsec / microBFD), do we need this protocol container?

Reply:
@apratimmukherjee do we need this container?  wouldn't macsec/microbfd be things run over the LAG, but not to signal the LAG?